packages/editor/src/browser/editor-command.ts

```diff
@@ -10,20 +10,21 @@ import { CommandContribution, CommandRegistry, Command } from "@theia/core/lib/c
 
 export namespace EditorCommands {
 
-    /**
-     * Show editor references
-     */
+    // Show editor references
     export const SHOW_REFERENCES: Command = {
         id: 'textEditor.commands.showReferences'
     };
-
+    export const CONFIG_TABBING: Command = {
+        id: 'textEditor.commands.configureTabbing',
```

Comment:
Would `textEditor.commands.configureIndentation` be a better name here? I'm actually not sure, @simark WDYT?

Reply:
You should add 2 commands more `Indent Using Spaces` and `Indent Using Tabs`. The user should be able to find them among other editor commands in the command palette without using the status bar.

`textEditor.configureIndentation` could be an internal command without a label as similar to `textEditor.commands.showReferences` which triggers the quick open width with indentation commands.